src/patterns/national-insurance-numbers/index.md.njk

```diff
@@ -30,6 +30,8 @@ If you currently use National Insurance numbers to verify identity, find out how
 
 Use a single [text input](/components/text-input/) labelled ‘National Insurance number’. Write it out in full and never use abbreviations such as ‘NINO’ or ‘NI Number’.
 
+When replaying the National Insurance number on something like [check your answers](/patterns/check-answers/) keep the format ‘QQ 12 34 56 C’ - the spaces will help break up the number for screen reader users.
```

Comment:
Thanks @htmlandbacon. I'd probably go "show" to make it a bit simpler and make it more generalised?

"Replay" might be a bit jargon, and the 'check answers' bit also begs the question about reformatting the user's answer... any idea if we have evidence whether it's okay to do this?

```suggestion
Show a National Insurance number using the format ‘QQ 12 34 56 C’ — the spaces will break up the number to make it easier to read, particularly for screen reader users.
```